invenio_app_ils/ui/src/invenio_app_ils/forms/components/SelectField.js

```diff
@@ -25,59 +24,44 @@ export class SelectField extends Component {
       : null;
   }
 
-  renderSelectField = props => {
+  renderFormField = props => {
     const {
       form: { values, setFieldValue, handleBlur, errors },
     } = props;
     return (
       <Form.Field>
-        <Form.Dropdown
-          id={this.fieldPath}
-          required={this.required}
+        <Form.Field
+          fluid
+          selection
+          control={Dropdown}
```

Comment:
Do we gain something with declaring it like that?

Reply:
Actually no we don't. When I tested it the first time it didn't respect the `searchInput` prop but it seems like it does. I'll change it back.